app/helpers/govuk_design_system_helper.rb

```diff
@@ -7,12 +7,20 @@
 require 'gds_design_system_breadcrumb_builder'
 
 module GovukDesignSystemHelper
-  def govuk_page_title(title = nil)
-    content_for :page_title, (title || contextual_title) + " - #{service_name} - GOV.UK"
+  def govuk_page_title(title = nil, caption = nil)
+    page_title_string = title || contextual_title
+    page_title_concat = "#{caption} #{page_title_string} - #{service_name} - GOV.UK".strip
```

Comment:
Could abstract these to their own private methods and call
```
content_for :page_title, page_title

...

private

def page_title(title, caption)
  _page_title = title || contextual_title
  _caption = caption.strip.concat(' ') if caption.present?
  "#{_caption}#{_page_title} - #{service_name} - GOV.UK".strip
end
```

Reply:
^^ this also handles the possibility that caption is no passed and prevents a whitespace before the page title